evalml/automl/utils.py

```diff
@@ -194,3 +196,26 @@ def get_best_sampler_for_data(X, y, sampler_method, sampler_balanced_ratio):
                 return "SMOTENC Oversampler"
         except ImportError:
             return "Undersampler"
+
+
+def get_hyperparameter_ranges(component_graph, custom_hyperparameters):
```

Comment:
Got it. And in this case, `component_graph` is a list of strings / component classes, right? As opposed to a `ComponentGraph` instance.

Could you please update the args list to indicate this? I know this isn't the official syntax but you could say "component_graph (list(str, class))"